app/base/models.py

```diff
@@ -85,24 +98,25 @@ class Property(db.Model):
     user_id = db.Column(db.ForeignKey("User.id"), nullable=False)
     users = db.relationship(User)
 
-    def __init__(self, name, desc, price, location, photos, user_id):
-        self.name = name
-        self.desc = desc
-        self.price = price
-        self.location = location
-        self.photos = photos
-        self.user_id = user_id
-
-    @property
-    def serialize(self):
-        return {
-            "name": self.name,
-            "desc": self.desc,
-            "price": self.price,
-            "location": self.location,
-            "photos": self.photos,
-            "user_id": self.user_id
-        }
+    # def __init__(self, name, desc, price, location, image_folder, photos, user_id):
+    #     self.name = name
+    #     self.desc = desc
+    #     self.price = price
+    #     self.location = location
+    #     self.photos = photos
+    #     self.image_folder = image_folder
+    #     self.user_id = user_id
+    #
+    # @property
+    # def serialize(self):
+    #     return {
+    #         "name": self.name,
+    #         "desc": self.desc,
+    #         "price": self.price,
+    #         "location": self.location,
+    #         "photos": self.photos,
+    #         "user_id": self.user_id
+    #     }
```

Comment:
did you get rid of this `code`

Reply:
Yes, I did. I figured out the same can be done using `propert_schema.dump`